multiaccounts/settings_notifications/database.go

```diff
@@ -0,0 +1,245 @@
+package notificationssettings
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type SettingType int
+
+const (
+	String SettingType = iota
+	Integer
+	Bool
+)
+
+// Columns' names
+const ColumnID = "id"
+const ColumnExemption = "exemption"
+const ColumnTextValue = "text_value"
+const ColumnIntValue = "int_value"
+const ColumnBoolValue = "bool_value"
+const ColumnExMuteAllMessages = "ex_mute_all_messages"
+const ColumnExPersonalMentions = "ex_personal_mentions"
+const ColumnExGlobalMentions = "ex_global_mentions"
+const ColumnExOtherMessages = "ex_other_messages"
```

Comment:
```suggestion
type Column string
const (
    ColumnID Column = "id"
    ColumnExemption = "exemption"
    ColumnTextValue = "text_value"
    ColumnIntValue = "int_value"
    ColumnBoolValue = "bool_value"
    ColumnExMuteAllMessages = "ex_mute_all_messages"
    ColumnExPersonalMentions = "ex_personal_mentions"
    ColumnExGlobalMentions = "ex_global_mentions"
    ColumnExOtherMessages = "ex_other_messages"
 )
```
Then, in functions like `buildSelectQuery(column string)`, you'd change the parameter name to `Column`. That way you guarantee that the functions will be called with the columns you intended and not with any random string